community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -621,6 +622,18 @@ private void createEmptyStore( File fileName, String typeAndVersionDescriptor, B
         }
     }
 
+    /**
+     * I.e. total number of used/unused records + 1
+     * @param storeFile {@link StoreFile} to get the name from.
+     * @param recordSize record size of that store.
+     * @return highId, i.e. an id one greater than the highest id in the store.
+     */
+    public long getHighId( StoreFile storeFile, int recordSize )
```

Comment:
Maybe it is worth making a comment/assertion that this method is only valid for opened store files that have no trailers?